### PR TITLE
added code to push images to jfrog

### PR DESCRIPTION
**Pull request by shubhamrai1993**

### CODE REVIEW

File: .github/workflows/update-artifacts.yml [hunk 1]
```diff
@@ -75,3 +81,26 @@ jobs:
...
+          with:
+            username: github-ci
+            password: ${{ secrets.JFROG_REGISTRY_PASSWORD }}
+            registry: tfy.jfrog.io
+
+        - name: Upload Artifacts
+          run: |
+            python scripts/upload-artifacts/upload_artifacts.py \
+              helm \
+              charts/tfy-k8s-aws-eks-inframold/artifacts-manifest.json \
```
Comment (DeeAjayi) [inline .github/workflows/update-artifacts.yml:99]: Won't using only one inframold chart exclude some images/charts that are not in tfy-k8s-aws-eks-inframold artifacts-manifest.json file?

File: scripts/upload-artifacts/tfy-agent-argo.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+apiVersion: argoproj.io/v1alpha1
```
Comment (DeeAjayi) [inline scripts/upload-artifacts/tfy-agent-argo.yaml:1]: Is there a reason for this being here?

Comment (DeeAjayi) [inline scripts/upload-artifacts/tfy-agent-argo.yaml:1]: Is there a reason for this being here?

File: .github/workflows/update-artifacts.yml [hunk 3]
```diff
@@ -75,3 +81,30 @@ jobs:
...
+            registry: tfy.jfrog.io
+
+        - name: Upload Artifacts
+          run: |
+            charts_list=("tfy-k8s-aws-eks-inframold tfy-k8s-azure-aks-inframold tfy-k8s-gcp-gke-standard-inframold tfy-k8s-generic-inframold")
+            for chart in $charts_list;
+            do
+              python scripts/upload-artifacts/upload_artifacts.py \
+                helm \
+                charts/tfy-k8s-aws-eks-inframold/artifacts-manifest.json \
```
Comment (innoavator) [inline .github/workflows/update-artifacts.yml:102]: charts/$chart

**Review by innoavator — COMMENTED** [review]
Can you rebase? the artifacts manifest changes should disappear then?

**Review by innoavator — APPROVED** [review]

**Pull request merged**